pf-codemods refuses to run: Cannot find module '@typescript-eslint/parser'

Tried running pf-codemods on my project, got the following output:

```
$ npx pf-codemods ./src
npx: installed 145 in 11.668s
Failed to load parser '@typescript-eslint/parser' declared in 'BaseConfig': Cannot find module '@typescript-eslint/parser'
Require stack:
- /home/elad/Projects/ocm/__placeholder__.js
```

--- a/packages/pf-codemods/index.js
+++ b/packages/pf-codemods/index.js
@@ -78,8 +78,7 @@ function runCodemods(path, otherPaths, options) {
     cache: true,
     cacheFile: '.eslintcache',
     cacheLocation: process.cwd(),
-    fix: options.fix,
-    resolvePluginsRelativeTo: __dirname
+    fix: options.fix
   });
   
   const report = engine.executeOnFiles(otherPaths.concat(path));

